modules/mobile/docs/schemas/Practitioner.yml

```diff
@@ -0,0 +1,21 @@
+type: object
+properties:
+  identifier:
+    type: array
+    items:
+      "$ref": ./AppointmentIdentifier.yml
+  name:
+    type: object
+    properties:
+      family:
+        type: string
+        description: Family name (surname).
+      given:
+        type: array
+        description: Given names (not always first). Given Names appear in the correct order for presenting the name.
+        items:
+          type: string
+    description: A human name.
```

Comment:
I know this was copied from somewhere else, but I appreciate specifying the species. 😂

Reply:
I'm just glad we won't have to re-factor everything when we discover alien life.